src/components/elements/Map-mapbox/MapControls/DeleteBulkPolygonsControl.tsx

```diff
@@ -0,0 +1,70 @@
+import { useT } from "@transifex/react";
+
+import { ModalId } from "@/components/extensive/Modal/ModalConst";
+import ModalDeleteBulkPolygons from "@/components/extensive/Modal/ModalDeleteBulkPolygons";
+import { useMapAreaContext } from "@/context/mapArea.provider";
+import { useModalContext } from "@/context/modal.provider";
+import { useSitePolygonData } from "@/context/sitePolygon.provider";
+import { SitePolygon } from "@/generated/apiSchemas";
+
+import Button from "../../Button/Button";
+import Text from "../../Text/Text";
```

Comment:
update the imports to not use relative imports

Reply:
done